src/views/Organisations/Organisation.vue

```diff
@@ -68,133 +58,146 @@
+                      <span v-if="organisation.alternativeNames.length > 0">
+                        <ul>
+                          <li
+                            v-for="(name, key) in organisation.alternativeNames"
+                            :key="'altname_' + key"
+                          >
+                            {{ name }}
+                          </li>
+                        </ul>
+                      </span>
+                      <span v-else> None </span>
+                    </v-list-item-content>
+                  </v-list-item>
+
+                  <v-list-item
+                    :key="'organisation_types'"
+                    class="body-1"
+                  >
+                    <v-list-item-content
+                      class="py-0 d-block"
+                    >
+                      <b class="blue--text">Types: </b>
+                      <span v-if="organisation.types.length > 0">
+                        {{ organisation.types | formatList }}
+                      </span>
+                      <span v-else> None </span>
+                    </v-list-item-content>
+                  </v-list-item>
+
+                  <!-- Parent organisations -->
+                  <v-list-item
+                    v-if="organisation.parentOrganisations"
+                    :key="'organisation_parents'"
+                    class="body-1"
                   >
-                    <b class="blue--text">Alternative Names: </b>
-                    <span v-if="organisation.alternativeNames.length > 0">
+                    <v-list-item-content
+                      class="py-0 d-block"
+                    >
+                      <b class="blue--text">This organisation belongs to: </b>
                       <ul>
                         <li
-                          v-for="(name, key) in organisation.alternativeNames"
-                          :key="'altname_' + key"
+                          v-for="(parent, key) in organisation.parentOrganisations"
+                          :key="'parent_' + key"
                         >
-                          {{ name }}
+                          <a
+                            target="_blank"
+                            @click="$router.push({path: '/organisations/' + parent.id})"
+                          >
+                            {{ parent.name }}
+                          </a>
                         </li>
                       </ul>
-                    </span>
-                    <span v-else> None </span>
-                  </v-list-item-content>
-                </v-list-item>
-
-                <v-list-item
-                  :key="'organisation_types'"
-                  class="body-1"
-                >
-                  <v-list-item-content
-                    class="py-0 d-block"
+                    </v-list-item-content>
+                  </v-list-item>
+
+                  <!-- Parent organisations -->
```

Comment:
can be changed into child organisation. its duplicate

Reply:
in Maintainer.vue an eslint issue spotted. 
![image](https://user-images.githubusercontent.com/25683981/111648840-b483a700-87fb-11eb-9ffe-60aa73c64d2c.png)